citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -99,6 +104,21 @@
         }
     });
 });
+
+$(document).ready(function() {
+    var queryParams = window.location.search;
+    if(queryParams)  {
+        const params = new URLSearchParams(queryParams);
+        params.delete('page');
+        params.delete('searchTerm');
```

Comment:
is there a reason these are deleted? if they exist, they need to be added to all links, dont' they? and the same goes for the above delete statements.

Reply:
We provide the new page number and also add the search term. They keep duplicating and probably have different values if we don't remove them.

Reply:
the searchterm would stay the same though, wouldn't it?

Reply:
The user could add new search term and search again. All the relevant parameters like page, collectionId, search term, etc. are added wherever necessary. We need to remove them from previous URI to avoid any ambiguity and duplication.

Reply:
but that's only for the search input field, isn't it? For everything else, the searchTerm would be the same? Just make sure to remove the searchterm when you build the search url.

Reply:
Previously when we didn't pass the parameters ahead, we already had the logic in place to add them to the URI with new values if necessary. This includes the searchTerm as well. So, the new URI will have those params included.